libr/anal/esil.c

```diff
@@ -2141,7 +2140,7 @@ static int esil_bigger(RAnalEsil *esil) { // 'dst > src' => 'src,dst,>'
 	}
 	free (dst);
 	free (src);
-	return true;
+	return ret;
 }
 
 static int esil_smaller_equal(RAnalEsil *esil) { // 'dst <= src' => 'src,dst,<='
```

Comment:
Agree but this probably could be done in another pr

> On 05 Jun 2016, at 21:01, Anton Kochkov <notifications@github.com> wrote:
> 
> In libr/anal/esil.c:
> 
> > @@ -2141,7 +2140,7 @@ static int esil_bigger(RAnalEsil *esil) { // 'dst > src' => 'src,dst,>'
> >  	}
> >  	free (dst);
> >  	free (src);
> > -	return true;
> > +	return ret;
> >  }
> >  
> >  static int esil_smaller_equal(RAnalEsil *esil) { // 'dst <= src' => 'src,dst,<='
> I think it's better to make those functions just return bool, not int.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>